### PR TITLE
Use label as pod name when not set

### PR DESCRIPTION
**Pull request by carlossg**
Use better labels in tests to identify runaway pods

### CODE REVIEW

File: src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java [hunk 1]
```diff
@@ -69,8 +69,8 @@ public void runInPod() throws Exception {
         List<PodTemplate> templates = cloud.getAllTemplates();
 
         PodTemplate template = null;
-        while ((template = podTemplateWithLabel("mypod", templates)) == null) {
-            LOGGER.log(Level.INFO, "Waiting for mypod template to be created");
+        while ((template = podTemplateWithLabel("runInPod", templates)) == null) {
```
Comment (Vlatombe) [inline src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesPipelineTest.java]: Could use a `TestName` rule to make it generic

**Comment by yrsurya** [comment]
Hi @carlossg  , I am trying to use a custom image with amazonlinux as base image, can you please let me know if that possible.

The default jnlp agent image used can be customized by adding it to the template

containerTemplate(name: 'jnlp', image: 'jenkins/jnlp-slave:3.10-1-alpine', args: '${computer.jnlpmac} ${computer.name}'),

I find this on documentation but stil it using alpine image.

much appreciated

**Comment by carlossg** [comment]
@yrsurya you should use jenkins-user mailing list for questions

**Comment by HerrmannHinz** [comment]
this means instead of `jenkins-slave-xxxx` the pods now will be called like `label-name-xxxx`? 
yay? yay!

**Pull request merged**